website/docs/d/cbr_rule.html.markdown

```diff
@@ -2,7 +2,7 @@
 layout: "ibm"
 page_title: "IBM : ibm_cbr_rule"
 description: |-
-  Get information about cbr_rule
```

Comment:
i thnk we shd have one tab here else subcategory will not be applicable

Reply:
fixed